app/src/main/java/cm/aptoide/pt/promotions/PromotionsAnalytics.java

```diff
@@ -132,4 +133,14 @@ public void sendPromotionsAppInteractInstallEvent(String packageName, float appc
           PROMOTIONS_INTERACT, AnalyticsManager.Action.CLICK, getViewName(true));
     }
   }
+
+  public void sendRefreshCaptchaEvent(String packageName) {
+    final Map<String, Object> data = new HashMap<>();
+    data.put(ACTION, "reload");
```

Comment:
I would extract this action "reload" as well and in order for it to be just like the others that are already extracted